### PR TITLE
Add code snippets for Swift subscribeToSessionUpdates(_:)

### PR DESCRIPTION
**Pull request by lowip**
# Pull Request/Issue Resolution

#### Description of Change:

Adding documentation to a new Swift SDK method, see https://github.com/braze-inc/swift-sdk/pull/620 and https://jira.braze.com/browse/SDK-4737

#### Is this change associated with a Braze feature/product release?
- [x] Yes, wait for next Swift SDK release
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [x] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [x] Check that all links work.
- [x] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @internetisaiah as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Maria Maldonado on the Legal team has signed off.
- [x] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @internetisaiah for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@internetisaiah</td>
    <td>Developer Guide<br>Technology partners<br>Channels</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>Frontend Infrastructure & Experience (FIX)</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/swift/analytics/tracking_sessions.md [hunk 1]
```diff
@@ -67,6 +67,52 @@ To detect sessions via your user, find your user on the dashboard and navigate t
...
+    default:
+      break;
+  }
+}];
+```
+
+{% endtab %}
+{% endtabs %}
+
+[1]: https://braze-inc.github.io/braze-swift-sdk/documentation/brazekit/braze/subscribetosessionupdates(_:)
```
Comment (lowip) [inline _docs/_developer_guide/platform_integration_guides/swift/analytics/tracking_sessions.md:115]: Link will work once next Swift SDK version is released

File: _docs/_developer_guide/platform_integration_guides/swift/analytics/tracking_sessions.md [hunk 2]
```diff
@@ -67,6 +67,52 @@ To detect sessions via your user, find your user on the dashboard and navigate t
...
 ![The app usage section of a user profile showing the number of sessions, last used date, and first used date.][session_tracking_7]
 
+## Subscribing to session updates
+
+The Braze SDK provides a [`subscribeToSessionUpdates(_:)`][1] method to listen for session updates:
+
+{% tabs %}
+{% tab swift %}
+
+```swift
```
Comment (jerielng) [inline _docs/_developer_guide/platform_integration_guides/swift/analytics/tracking_sessions.md:77]: Optional: We could also call out the stream version if we want to document both options. We have a similar example with the Content Cards subscription [here](https://www.braze.com/docs/developer_guide/customization_guides/content_cards/logging_analytics/?tab=ios#listening-for-card-updates).

**Comment by josh-mccrowell-braze** [comment]
Tagging @internetisaiah just for visibility/shepherding once the PR is complete

File: _docs/_developer_guide/platform_integration_guides/swift/analytics/tracking_sessions.md [hunk 3]
```diff
@@ -67,6 +67,52 @@ To detect sessions via your user, find your user on the dashboard and navigate t
...
+// This subscription is maintained through a Braze cancellable, which will observe for changes until the subscription is cancelled.
```
Comment (jacksonemiller) [inline _docs/_developer_guide/platform_integration_guides/swift/analytics/tracking_sessions.md:78]:
```
...which will observe for changes...
```
How about either "will watch for changes" or "will observe changes" here?

**Review by jacksonemiller — APPROVED** [review]

**Review by vanessaland — APPROVED** [review]

**Review by hokstuff — APPROVED** [review]
LGTM

**Comment by internetisaiah** [comment]
**Note:** I added the "do not merge" tag to this PR until it's ready for review from the docs team. Just ping me when you're ready for me to look over your work @lowip ! 👍🏽

**Comment by lowip** [comment]
@internetisaiah this feature has been released, please take a look when you have time :)

**Review by internetisaiah — APPROVED** [review]
LGTM

**Pull request merged**